tools/eve-gen-single-use-installer.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/sh
+# shellcheck disable=SC2086
+# shellcheck disable=SC2181
+
+USAGE="Usage: $0 [options]
+Options:
+  -c <controller's fully-qualified domain name>
+  -t <EVE image tag>
+     Production EVE images built by lf-edge use tags with the format:
+       <major>.<minor>.<bugfix>-<hypervisor>-<architecture>
+     Where <hypervisor> is one of: kvm | xen
+     And <architecture> is one of: amd64 | arm64 | riscv64
+     Mandatory argument.
+  -s <soft serial string>
+  -e <path to onboarding certificate>
+     The certificate should be in the PEM format.
+  -k <path to onboarding key>
+     The key should be in the PEM format.
+  -b <path to bootstrap config>
+     The file should contain BootstrapConfig (see EVE API) serialized with protobuf binary encoding.
```

Comment:
Should we say something about default for these? in the usage message?

Reply:
Done, described default values for all optional arguments.